bdp_app/pubspec.yaml

```diff
@@ -28,9 +28,17 @@ dependencies:
   webview_flutter: 0.3.15+1
 
 dev_dependencies:
+
   flutter_test:
     sdk: flutter
 
+  flutter_launcher_icons: "^0.7.4"
+
+flutter_icons:
+  android: "launcher_icon"
+  ios: true
+  image_path: "/home/marian/Schreibtisch/BDP-Logo_250x188.png"
```

Comment:
Das wird nicht funktionieren - du musst die Datei ins GitHub-Repo hochladen.

Reply:
Wenn ich das richtig verstanden habe, Ist das nur zum einmaligen laden der Datei. Unter /bdp_app/android/app/src/main/res müsste das icon abgespeichert sein1

Reply:
Ok - dann ist muss hier der Pfad "bdp_app/android/app/src/main/res/..." angegeben sein. In den Dateien, die auf github hochgeladen werden, dürfen keine Pfade zu lokalen Dateien mehr existieren.

Reply:
@mr-ml normalerweise sollte der eine conversation schließen, der sie auch aufgemacht hat. Und davon unabhängig hat Shuki schon recht, hier sollte kein Verweis auf dein lokales Dateisystem stehen.